client/src/layouts/Default.js

```diff
@@ -16,35 +18,73 @@ const header = {
 
 // Used to add new pages to the navbar
 const routes = {
-    "Home": {
+    "/": {
+        name: "Home",
         thumbnail: "https://wallpapercave.com/wp/wp6092911.jpg",
-        component: <Home />
+        component: <Home />,
+        displayInNav: true,
+        displayParallax: true
     },
-    "About": {
+    "/about": {
+        name: "About",
         thumbnail: "https://a-static.besthdwallpaper.com/australian-shepherd-dog-wallpaper-1920x1280-54743_38.jpg",
-        component: <About />
+        component: <About />,
+        displayInNav: true,
+        displayParallax: true
     },
-    "Login": {
+    "/login": {
+        name: "Login",
         thumbnail: "https://a-static.besthdwallpaper.com/australian-shepherd-dog-wallpaper-1920x1280-54743_38.jpg",
-        component: <Login />
+        component: <Login />,
+        displayInNav: true,
+        displayParallax: false
     },
-    "Signup": {
+    "/signup": {
+        name: "Signup",
         thumbnail: "https://a-static.besthdwallpaper.com/australian-shepherd-dog-wallpaper-1920x1280-54743_38.jpg",
-        component: <Signup />
+        component: <Signup />,
+        displayInNav: true,
+        displayParallax: false
     },
+    "/404": {
+        name: "404",
+        thumbnail: "https://atlassianblog.wpengine.com/wp-content/uploads/2017/12/44-incredible-404-error-pages@3x.png",
+        component: <NotFound/>,
+        displayInNav: false,
```

Comment:
Because displayInNav and displayParallax are set to false, we wont see them displayed when using that componet.